docs/reference/esql/esql-syntax.asciidoc

```diff
@@ -160,14 +160,15 @@ Datetime intervals and timespans can be expressed using timespan literals.
 Timespan literals are a combination of a number and a qualifier. These
 qualifiers are supported:
 
-* `millisecond`/`milliseconds`
-* `second`/`seconds`
-* `minute`/`minutes`
-* `hour`/`hours`
-* `day`/`days`
-* `week`/`weeks`
-* `month`/`months`
-* `year`/`years`
+* `millisecond`/`milliseconds`/`ms`
+* `second`/`seconds`/`s`
```

Comment:
Missing `sec`
```suggestion
* `second`/`seconds`/`sec`/`s`
```

Reply:
Yeh... I guess if we opened the gates, "sec" can be there too. I'll add it.